components/bounty-status-info.tsx

```diff
@@ -8,40 +8,51 @@ interface IBountyStatusInfo {
   issueState: IssueState;
 }
 export default function BountyStatusInfo({ issueState }: IBountyStatusInfo) {
-  function handleColorState() {
+  function handleColorState(ellipse = false) {
     switch (issueState?.toLowerCase()) {
     case "draft": {
-      return "bg-white-50";
+      if(ellipse) return "bg-info"
+      return "bg-info-30 text-info";
     }
     case "open": {
-      return "bg-primary text-white";
-    }
-    case "in progress": {
-      return "bg-primary text-white";
+      if(ellipse) return "bg-success"
+      return "bg-success-30 text-success";
     }
     case "canceled": {
-      return "bg-dark-gray text-white";
+      if(ellipse) return null
+      return "bg-danger-30 text-danger-70";
     }
     case "closed": {
-      return "bg-dark-gray text-white";
+      if(ellipse) return null
+      return "bg-dark-gray text-white-40";
     }
     case "ready": {
-      return "bg-success";
+      if(ellipse) return "bg-success"
+      return "bg-success-30 text-success";
     }
-    case "done": {
-      return "bg-success";
+    case "proposal": {
+      if(ellipse) return "bg-purple"
+      return "bg-purple-30 text-purple";
     }
-    case "disputed": {
-      return "bg-danger text-white";
+    case "funding": {
+      if(ellipse) return "bg-light-warning"
+      return "bg-light-warning-30 text-light-warning";
     }
     default: {
-      return "primary";
+      if(ellipse) return "bg-primary"
+      return "bg-primary-30 text-white";
     }
     }
   }
 
+  function renderEllipse(issueState: IssueState) {
+    if(["canceled", "closed"].includes(issueState)) return null
+    return <div className={`ellipse bg-primary me-2 ${handleColorState(true)}`} />
+  }
+
   return (
-    <span className={`status caption-small ${handleColorState()}`}>
+    <span className={`d-flex status caption-medium py-1 px-3 ${handleColorState()}`}>
```

Comment:
This can be simplified via constants (because they are constants, and we don't really need logical getters),

```tsx
const Colors = {
  open: { ellipse: 'bg-success', badge: 'bg-success-30 text-success' },
  closed: { ellipse: '', badge: 'bg-dark-gray text-white-40' }
  // ...
}
  return <Badge bg="" className="Colors[issueState].badge">
    { (Colors[issueState].badge && <div className={`ellipse bg-primary me-2 ${Colors[issueState].badge}`} /> }
    {issueState && <Translation ns="bounty" label={`status.${issueState}`} />}
  </Badge>
```